templates/http/base/deployment-database.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:  
```

Comment:
Nit: tad annotations probably aren't necessary

Reply:
@johnmcollier could you PTAL